### ISSUE TITLE
WCAG conformance requirements - Levels A / AA /  AAA

### ISSUE THREAD

**Issue by LaZay** [w3c/publ-a11y#141]
In our born accessible EPUBs, we need to address every WCAG level, including AAA (metadata + content structure).

The [WCAG note ](https://www.w3.org/TR/WCAG21/#cc1) that is pointed out by the [EPUB A11Y spec](https://www.w3.org/TR/epub-a11y-11/) only says:
_It is not recommended that Level AAA conformance be required as a **general policy** for entire sites because it is not possible to satisfy all Level AAA Success Criteria for **some content** ._

 If the Web site / the EPUB does not countain any such content, the AAA level can be achieved.

Furthermore, if Education innovative publishers / pure players decided to offer WCAG AAA EPUBs to answer a need for fully distant teaching, with EPUB technology instead of LMS (eLaerning Management System) technology, they must be able to signal their little jewels in the OPF!

There is another [WCAG note](https://www.w3.org/TR/WCAG22/#wcag-2-layers-of-guidance) of high interest.
_Note that even content that conforms at the highest level (AAA) will not be accessible to individuals with all types, degrees, or combinations of disability, particularly in the cognitive, language, and learning areas._ 
Absolutely! Let's keep this in mind, which is very different from AAA level exclusion though.

[This EPUB A11Y spec ](https://www.w3.org/TR/epub-a11y-11/#sec-wcag-conf)note sounds somewhat excessive, and misleading.
_Although, as [noted in WCAG](https://www.w3.org/TR/WCAG21/#h-note-29), conforming at level AAA is **typically not possible**, and not required by this specification, [EPUB creators](https://www.w3.org/TR/epub/#dfn-epub-creator) are encouraged to follow the practices detailed in AAA success criteria when producing accessible EPUB publications._

Furthermore, I still do not understand why a spec about EPUB a11y should make any choice on WCAG level (according European legislation). All born accessible EPUB creators are not Europeans as far as I know... Better starting with born accessible EPUBs level A, than not starting born accessible EPUBs at all. That is the risk when the bar is set too high too fast. That is why WCAG defined 3 levels, to encourage accessbility development. Not to discourage it right away. But this is another issue.

**Comment by mattgarrish** [comment]
The Accessibility specification doesn't say you can't meet AAA, but it is typically difficult if only because of SC 3.1.5:

> When text requires reading ability more advanced than the [lower secondary education level](https://www.w3.org/TR/WCAG22/#dfn-lower-secondary-education-level) after removal of proper names and titles, [supplemental content](https://www.w3.org/TR/WCAG22/#dfn-supplementary-content), or a version that does not require reading ability more advanced than the lower secondary education level, is available.

That makes most materials not written for an audience with an education level in the grade 7-9 range or lower (as best I understand that classification) a failure.

At the same time, it doesn't bother me if we remove the editorializing from the note so it only reads:

> Although conforming at level AAA is not required by this specification, [EPUB creators](https://www.w3.org/TR/epub/#dfn-epub-creator) are encouraged to follow the practices detailed in AAA success criteria when producing accessible EPUB publications.

But we need to move fast to change it.

**Comment by LaZay** [comment]
Your proposal sounds perfect!

**Comment by clapierre** [comment]
I am fine with revising this as Matt suggests.

**Comment by LaZay** [comment]
Thanks for your returns.

Just to let you know, I created another ticket for the WCAG.
[https://github.com/w3c/wcag/issues/2852](https://github.com/w3c/wcag/issues/2852)
Even Alex Bernier from Braillenet considered that this SC was "too much". I forgot about this.
He who tries nothing has nothing.

**Comment by gautierchomel** [comment]
@LaZay can we consider this issue addressed and close it?

**Comment by LaZay** [comment]
Yes, the correction has been integrated in [version 1.1 of the EPUB Accessibility specification](https://www.w3.org/TR/epub-a11y-11/#sec-wcag-conf).
Thanks!
